app/main.py

```diff
@@ -1,6 +1,38 @@
+import requests
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+URL = "http://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
+API_KEY = os.environ["API_KEY"]
```

Comment:
if the env variable is not found, your code will break

Reply:
ok, I write key in python file